src/app/info/components/GovernanceCharts.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import ChartGovernanceAvgVotes from "@/app/info/components/ChartGovernanceAvgVotes";
+import ChartGovernanceRequiredDelegates from "@/app/info/components/ChartGovernanceRequiredDelegates";
+import ChartGovernanceVotableSupply from "@/app/info/components/ChartGovernanceVotableSupply";
+import ChartGovernanceTopDelegates from "@/app/info/components/ChartGovernanceTopDelegates";
+import ChartGovernanceActiveDelegates from "@/app/info/components/ChartGovernanceActiveDelegates";
+
+interface TabProps {
+  value: string;
+  title: string;
+  description: string;
+}
+
+const TabTrigger: React.FC<TabProps> = ({ value, title, description }) => {
+  return (
+    <TabsTrigger
+      value={value}
+      className="flex flex-col last:col-span-2  last:sm:col-span-1  opacity-100 border !border-gray-300 border-t-0 border-l-0 first:rounded-tl-lg last:rounded-tr-lg  bg-gray-fa data-[state=active]:bg-white p-4 sm:p-8 w-full"
+    >
+      <h3 className="text-sm sm:text-base font-semibold text-black">{title}</h3>
+      <p className="text-xs font-medium text-gray-4f">{description}</p>
+    </TabsTrigger>
+  );
+};
+
+interface GovernanceChartsProps {
+  getMetrics: (metric: string, frequency: string) => Promise<any>;
+  getVotes: () => Promise<any>;
+  getDelegates: () => Promise<any>;
+}
+
+const GovernanceCharts = ({
+  getMetrics,
+  getVotes,
+  getDelegates,
+}: GovernanceChartsProps) => {
+  const tabs = [
+    {
+      value: "top-delegates",
+      title: "Top Delegates",
+      description: "",
```

Comment:
does it require a description? Just feels weird to have this empty

Reply:
@mcgingras ah yes, we used to have description placeholder in the UI but there was no easy way to get them from the back end. I am going to remove them for now.